src/compiler/transformers/static-to-meta/component.ts

```diff
@@ -18,14 +18,28 @@ import { parseStringLiteral } from './string-literal';
 import { setComponentBuildConditionals } from '../component-build-conditionals';
 import ts from 'typescript';
 
+/**
+ * Given an instance of typescript's IR for a class decoration ({@see ts.ClassDeclaration})
+ * which represents a Stencil component class declaration, parse and format
+ * various pieces of data about static class members which we use in the
+ * compilation process
+ *
+ * @param compilerCtx the current compiler context
+ * @param typeChecker a typescript typechecker instance
+ * @param cmpNode the typescript class declaration for the component
+ * @param moduleFile stencil's IR for a module, used here as an out param
+ * @param transformOpts options which control various aspects of the
+ * transformation
+ * @returns the TypeScript class declaration instance with which the function
+ * was called
+ */
 export const parseStaticComponentMeta = (
   compilerCtx: d.CompilerCtx,
   typeChecker: ts.TypeChecker,
   cmpNode: ts.ClassDeclaration,
   moduleFile: d.Module,
-  nodeMap: d.NodeMap,
```

Comment:
the `nodeMap` is already present on `CompilerCtx` and at every call site this function was being called with the `nodeMap` set on the same `CompilerCtx` object already being passed to it, so this argument was redundant and I therefore decided to remove it